src/components/ContentHighlights/HighlightStepper/ContentHighlightStepper.jsx

```diff
@@ -95,103 +106,149 @@ const ContentHighlightStepper = ({ enterpriseId }) => {
       setIsPublishing(false);
     }
   };
+  const closeStepper = () => {
+    open();
+  };
+
+  /* Start beforeunload event */
+  // This section triggers browser response to unsaved items when the stepper modal is open/active
+  useEffect(() => {
+    const preventUnload = () => {
+      /* eslint-disable no-restricted-globals */
+      /* Mandatory requirements to trigger response by browser, event.preventDefault && event.returnValue
+      A return value is required to trigger the browser unsaved data blocking modal response */
+      event.preventDefault();
```

Comment:
Where is `event` coming from here? Does it need to be passed as the function argument to `preventUnload`? E.g., `const preventUnload = (e) => { ... }`. This may make it possible to remove the above `eslint-disable no-restricted-globals`, as you'll no longer be referencing the global `event`.